tests/robot/COMPOSITION_TESTS/VALIDATION/V.1_Validation_test_of_COMPOSITION_class.robot

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) 2019 Wladislaw Wagner (Vitasystems GmbH), Pablo Pazos (Hannover Medical School),
+# Nataliya Flusman (Solit Clouds), Nikita Danilin (Solit Clouds)
+#
+# This file is part of Project EHRbase
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+*** Settings ***
+Documentation   Composition Integration Tests
+Metadata        TOP_TEST_SUITE    COMPOSITION
+Resource        ../../_resources/keywords/composition_keywords.robot
+
+Suite Setup     Precondition
+Suite Teardown  Postcondition
+
+Force Tags      future
+
+
+*** Variables ***
+${COMPOSITIONS_PATH}    ${EXECDIR}/robot/_resources/test_data_sets/compositions/CANONICAL_JSON
+
+
+*** Test Cases ***
+
+Validation test of COMPOSITION class
+    [Template]    Validate of compositions
+
+    #language    territory    category    composer    status_code
```

Comment:
Hi, @wlad! I'm answering this question if you don't mind. First, if we add this combination it'll be excess because some combinations are repeating ourselves (in meaning). Second, our test was created by using the test design method (pair). it had been done to simplify the autotest itself.